qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -84,6 +90,11 @@ function cleanup()
         fi
     done
     rbd namespace remove ${POOL}/${NS}
+
+    # cleanup/reset default pool
+    ceph osd pool delete ${ANOTHER_POOL} ${ANOTHER_POOL} --yes-i-really-really-mean-it
+    rbd config global set global rbd_default_pool ${POOL}
+    rbd config global get global rbd_default_pool | grep "^${POOL}\\b"
```

Comment:
Nit: this "get" confirmation is probably unnecessary as `cleanup` is best effort.